cdap-ui/app/directives/my-pipeline-scheduler/my-pipeline-scheduler.html

```diff
@@ -0,0 +1,418 @@
+<!--
+  Copyright © 2017 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<div class="pipeline-scheduler-content">
+  <div class="pipeline-scheduler-header">
+    <h3 class="modeless-title">
+      Schedule for
+      <span ng-if="SchedulerCtrl.pipelineName.length > 0">
+        "{{SchedulerCtrl.pipelineName}}"
+      </span>
+    </h3>
+    <div class="btn-group">
+      <a class="btn" ng-click="SchedulerCtrl.onClose()">
+        <span class="fa fa-remove"></span>
+      </a>
+    </div>
+  </div>
+  <div class="pipeline-scheduler-body">
+    <div class="schedule-content">
+      <fieldset ng-disabled="SchedulerCtrl._isDisabled">
+        <div class="schedule-types">
```

Comment:
should add a `row` class here. `col-*` classes are using `float: left`, so it needs to the clearfix, especially when the total number of column in the row is less than 12